source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixture.cs

```diff
@@ -271,7 +271,7 @@ public void AuthorisingWithAmazonAccount()
             variables.Set("Octopus.Action.Kubernetes.CertificateAuthority", certificateAuthority);
             variables.Set($"{certificateAuthority}.CertificatePem", eksClusterCaCertificate);
             var wrapper = CreateWrapper();
-            TestScript(wrapper, "Test-Script");
+            TestScript(wrapper, "Test-Script", variables.Get("Octopus.Action.Kubernetes.CustomKubectlExecutable"));
```

Comment:
I think we need to be checking if this exists, otherwise what would it do?

Reply:
I don't think that it will ever not be set because it's set in the `[SetUp]` method (line 78). If it's not set then there is something wrong with the test and we would expect it to fail, wouldn't we?

Reply:
I guess I could make it throw if it's not set with a nicer message.

Reply:
I agree with what you are saying but it is about helping someone in the future in my mind 😄 Maybe a comment about why we are doing this wouldn't hurt either

Reply:
Yep good point - I've updated it now 👍